### PR TITLE
feats: 1-event cards list 2-header routes 3-translation 4-old translation enhanced

### PR DESCRIPTION
**Pull request by whereismyname7**
…ion enhanced

### CODE REVIEW

**Review by bayanbamb — APPROVED** [review]
Great job!

File: src/app/components/events/events-card-list/events-card-list.component.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.card{
+    border: 1;
+    box-shadow: 0px 0px 25px 0px #0000001A;
+    color: #264653;
+    width: 30%;
+    font-family: Neo Sans;
```
Comment (MohammadWS) [inline src/app/components/events/events-card-list/events-card-list.component.css]: use Neo Sans Arabic as it is used by other components for more consistency

File: src/assets/i18n/en.json
Comment (MohammadWS) [inline src/assets/i18n/en.json:1]: if you remove the "on-site" from ar.json, i think it's better remove also from here
so as the removal of 'remote'

**Review by MohammadWS — COMMENTED** [review]
look good!

**Pull request merged**